tests/data/test_maestro.py

```diff
@@ -137,23 +136,24 @@ def test_maestro_invalid_tracks(tmpdir: str) -> None:
             (
                 getattr(splits, split)
                 | f"Write {split} to text"
-                >> beam.io.WriteToText(os.path.join(tmpdir, f"output_{split}.txt"), shard_name_template="")
+                >> beam.io.WriteToText(str(tmp_path / f"output_{split}.txt"), shard_name_template="")
             )
 
     for track_id, split in input_data:
-        with open(os.path.join(tmpdir, f"output_{split}.txt"), "r") as fp:
+        with open(str(tmp_path / f"output_{split}.txt"), "r") as fp:
```

Comment:
If I recall cor
```suggestion
        with open(tmp_path / f"output_{split}.txt", "r") as fp:
```
open can take a path.